chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -97,7 +97,10 @@
     @Autowired private MessageSource messageSource;
 
     @ApiOperation(value = "Get auditable data for certification bodies.",
-            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
+            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results. "
+                    + "Security Restrictions: ROLE_ADMIN AND ROLE_ONC_ADMIN may set the 'showDeleted' flag to true "
```

Comment:
I thought we got rid of the `showDeleted` flags?

Reply:
We did and I got rid of the showDeleted documentation at that time.

Reply:
I was simply cutting and pasting the security restrictions from the spreadhseet.  The code is correct and the  Swagger annotations should be correct now.